screens/map/MapScreen.js

```diff
@@ -16,13 +17,14 @@ import BottomSheet from 'reanimated-bottom-sheet';
 import Hamburger from '../../components/Hamburger';
 import StoreProducts from '../../components/product/StoreProducts';
 import { getProductData, getStoreData } from '../../lib/mapUtils';
-import { Subtitle } from '../../styled/shared';
+import { Body, ButtonLabel } from '../../components/BaseComponents';
 import {
   SearchBar,
-  StoreModal,
-  StoreModalBar,
-  TopText
+  BottomSheetContainer,
+  BottomSheetHeaderContainer,
+  StoreModalBar
```

Comment:
this is a nit but try to use hanging commas so future diffs won't include extra lines